code/provision/playbooks/terraform-state-server-create.yml

```diff
@@ -36,17 +28,17 @@
       os_server:
         state: present
         name: datalabs-terraform-state
-        image: ubuntu-1604-20180518
-        key_name: llogr-c9834a39ddef2dd489d7bd976d0ac79c
+        image: "{{ server_image }}"
+        key_name: "{{ key_name }}"
         timeout: 200
         flavor: 12
```

Comment:
So the VM size be hard coded?

Reply:
For the state server I think that is fine.